### DISCUSSION TITLE
How to scan using the http protocol with httpx turned off

### DISCUSSION THREAD

**Discussion by invoke888** [projectdiscovery/nuclei#5943] [Q&A]
In the large scale scanning due to the excessive number of ip, it is a waste of time to turn on httpx, so I want to turn off httpx, so I use the -nh parameter, but nuclei will default to use https protocol for scanning, how can I specify nuclei to use http protocol for scanning?

**Comment by ehsandeep** [comment]
@invoke888 httpx handles the default scheme to use, if you are looking to disable that option, just add http scheme appened in your input list to scan without probing.

**Reply by invoke888** [reply]
I see what you mean, but my input file is something like 192.168.0.1/16 meaning I have to write a script to extract each ip and add “http://”? Is there a faster way for me to do this?
